.gitignore

```diff
@@ -11,5 +11,7 @@
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
+.idea/
```

Comment:
Since everyone uses different development software, it would make more sense to have this in your global gitignore than in the individual repositories.

Reply:
previously unware of global gitignore ... trying it out now